pkg/plugins/autodiscovery/dockercompose/compose.go

```diff
@@ -0,0 +1,202 @@
+package dockercompose
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"github.com/updatecli/updatecli/pkg/core/config"
+	"github.com/updatecli/updatecli/pkg/core/pipeline/resource"
+	"github.com/updatecli/updatecli/pkg/core/pipeline/source"
+	"github.com/updatecli/updatecli/pkg/core/pipeline/target"
+	"github.com/updatecli/updatecli/pkg/core/transformer"
+	"github.com/updatecli/updatecli/pkg/plugins/resources/dockerimage"
+	"github.com/updatecli/updatecli/pkg/plugins/resources/yaml"
+)
+
+var (
+	// DefaultFileMatch specifies the file name patterns identifying Docker Compose files.
+	DefaultFileMatch []string = []string{
+		"docker-compose.yaml",
+		"docker-compose.*.yaml",
+		"docker-compose.yml",
+		"docker-compose.*.yml",
+	}
+)
```

Comment:
```suggestion
const (
	// DefaultFileMatch specifies the default file shell pattern to identify Docker Compose files
	// Ref. https://pkg.go.dev/path/filepath#Match and https://go.dev/play/p/y2b7tt03r8Q to test
	DefaultFilePattern string = "*docker-compose*.y*ml"
)
```

Using a const is more efficient and easier to use. Also added a go playground example to help future maintainers